assets/js/components/ExecutionResults/ExecutionResults.test.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { screen } from '@testing-library/react';
-
+import '@testing-library/jest-dom/extend-expect';
```

Comment:
Could you move this to the bottom of these imports?

Reply:
Sure :zap: